internal/ospkg/archlinux.go

```diff
@@ -9,25 +23,25 @@ import (
 	"path"
 	"strings"
 
-	"github.com/virtual-kubelet/systemk/pkg/unit"
+	"github.com/virtual-kubelet/systemk/internal/unit"
```

Comment:
yes `internal` exists as a thing, but now I need to `cd` once more. Can we remove `internal`?